clients/js/genjs.go

```diff
@@ -174,6 +191,17 @@ class {{.ClassName}} {
    * in milliseconds. This can be overridden on a per-request basis.
    * @param {module:{{.ServiceName}}.RetryPolicies} [options.retryPolicy=RetryPolicies.Single] - The logic to
    * determine which requests to retry, as well as how many times to retry.
+   * @param {Object} [options.circuit] - Options for constructing the client's circuit breaker.
+   * @param {bool} [options.circuit.debug] - When set to true will not circuit break. Default: true.
```

Comment:
I am passing this setting directly through to the `circuitBreakerForceClosed` setting, so maybe it could be named `forceClosed`?